### ISSUE TITLE
normalization

### ISSUE THREAD

**Issue by SeunghyunSEO** [SeunghyunSEO/SeunghyunSEO.github.io#28]
![batch_norm2](https://user-images.githubusercontent.com/48202736/108725143-69160a00-7569-11eb-90ca-bc55faa7a0fc.jpeg)
![batchnorm1](https://user-images.githubusercontent.com/48202736/108725152-6ca99100-7569-11eb-8131-1ee223119619.jpeg)
![norm1](https://user-images.githubusercontent.com/48202736/108725156-6d422780-7569-11eb-82eb-06aad5c2b647.png)
![norm2](https://user-images.githubusercontent.com/48202736/108725160-6d422780-7569-11eb-8303-f2c5088f8699.png)
![norm3](https://user-images.githubusercontent.com/48202736/108725162-6ddabe00-7569-11eb-8e12-3f43f58066ed.png)
